src/workerd/server/tests/python/workflow-entrypoint/workflow.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2025 Cloudflare, Inc.
+# Licensed under the Apache 2.0 license found in the LICENSE file or at:
+#     https://opensource.org/licenses/Apache-2.0
+
+from workers import NonRetryableError, WorkflowEntrypoint
```

Comment:
In JavaScript, we import `NonRetryableError` from `workflows`

Not the end of the world but consistency would be nice